azure-toolkit-libs/azure-toolkit-common-lib/src/main/java/com/microsoft/azure/toolkit/lib/common/bundle/AzureString.java

```diff
@@ -47,6 +46,11 @@ public String getString() {
     }
 
     public String getString(Object... params) {
+        // sometimes we may need to print an empty line by: messager.info("")
+        // we need to
+        if (StringUtils.isBlank(name)) {
+            return "";
+        }
```

Comment:
This is not the "right" way to print a empty line.
And I think this issue is not important/severe enough to be fixed as a hotfix